docs/tfengine/README.md

```diff
@@ -53,11 +53,11 @@ Use our [example](../../examples/tfengine) configs to quickly get started.
 
 1. Set up your
     [organization](https://cloud.google.com/resource-manager/docs/creating-managing-organization)
-    for Google Cloud resources and [G Suite Domain](https://gsuite.google.com/) for
+    for Google Cloud resources and either [G Suite Domain](https://gsuite.google.com/) or IAM for
```

Comment:
This step is to create the gsuite domain and groups so later they can be used in IAM settings. I'm not sure if adding IAM here is applicable.

Reply:
or you can say

"... and [G Suite Domain](https://gsuite.google.com/) for groups that later be used in Cloud IAM."

Reply:
You can use IAM without gsuite.

Reply:
Are you using an googlegroup or a google account (e.g. gmail) instead?

Reply:
I created the organization as a company with my domain name and then went to IAM and created the groups.   I didn't need gsuite so never created a gsuite domain

An Organization resource is available for G Suite and Cloud Identity customers:

G Suite: Learn About G Suite.
Cloud Identity: Learn About Cloud Identity.
Once you have created your G Suite or Cloud Identity account and associated it with a domain, your organization resource will be automatically created for you. The resource will be provisioned at different times depending on your account status:

Reply:
Thanks for the catch! I admit when I was initially writng this guide I was more restrictive than is actually necessary.

Maybe we can reword it to something like 

1. [Setup your organization](...) for Google Cloud resources.

1. [Create](https://support.google.com/a/answer/33343?hl=en) the following administrative [IAM](https://cloud.google.com/iam/docs/overview#concepts_related_identity) groups:
  ...